content/docs/setup/kubernetes/upgrading-istio.md

```diff
@@ -55,20 +53,19 @@ or
 $ kubectl apply -f install/kubernetes/istio-auth.yaml
 ```
 
-The rolling update process will upgrade all deployments and configmaps to the new version. After this process finishes, your Istio control plane should be updated to the new version and the control plane itself should be using the new Envoy v2 proxy. Your existing application should continue to work without any change, using the Envoy v1 proxy and the v1alpha1 route rules.  In any case if there is any critical issue with the new control plane, you can simply rollback the changes either by applying the old version yaml files.
+The rolling update process will upgrade all deployments and configmaps to the new version. After this process finishes, your Istio control plane should be updated to the new version. Your existing application should continue to work without any change, using the Envoy v1 proxy and the v1alpha1 route rules. If there is any critical issue with the new control plane, you can rollback the changes by applying the yaml files from the old version.
 
 ### Sidecar upgrade
 
-After the control plane is upgraded, you will need to re-inject the new version
-of sidecar proxy. There are two cases: Manual injection and Automatic injection.
+The applications already running istio will still using the sidecar from 0.7.1 and will continue to work. After the control plane is upgraded, you will need to re-inject so they run with the new version of sidecar proxy. There are two cases: Manual injection and Automatic injection.
 
 1.  Manual injection:
 
     If automatic sidecar injection is not enabled, you can upgrade the
     sidecar manually by running the following command:
 
     ```command
-    $ kubectl apply -f <(istioctl kube-inject -i $ISTIO_NAMESPACE -f $ORIGINAL_DEPLOYMENT_YAML)
+    $ kubectl apply -f <(istioctl kube-inject -f $ORIGINAL_DEPLOYMENT_YAML)
```

Comment:
shouldn't the user specify proxyv2 on the inject cmd because the default is proxyv1 in this case?

Reply:
That would be for migration from alpha1 to alpha3 - I think the doc at this point is about users who want to keep using alpha1/v1 and upgrade to 0.8. They will still use proxyv1.